Front/app/ns_modules/ns_grid/grid.view.js

```diff
@@ -78,7 +78,7 @@ define([
         onGridReady: function(){
           $.when(_this.deferred).then(function(){
             setTimeout(function(){
-              _this.gridOptions.api.sizeColumnsToFit();
+              //_this.gridOptions.api.sizeColumnsToFit();
```

Comment:
Don't comment it. It's used somewhere else. I don't have rights on your branch.
Column's size must be defined in base, then auto fit from those sizes.